src/software/proto/message_translation/protobuf_message_translator.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "shared/proto/geometry.pb.h"
+#include "shared/proto/tbots_software_msgs.pb.h"
+#include "shared/proto/vision.pb.h"
+#include "software/primitive/primitive.h"
+#include "software/world/world.h"
+
+/**
+ * Translates internal messages such as World, Geometry and Primitives to their
+ * respective protobuf message.
+ */
+class ProtobufMessageTranslator
```

Comment:
Any reason these can't be free functions since they're just static functions of this class with no member variables.

Reply:
yup thats true, the class is unnecessary, turned it into a namespace

Reply:
Do we even need the namespace? Probably worth a talk about how we want to do namespace since ages (ie. a year?) ago we sort of decided they weren't worth it since we don't need to "protect" our code or other people's code from conflicts since we "own" everything, and they _can_ cause headaches for newer people.

Obviously we aren't consistent currently and a few things still have them but worth making the call now how we want to proceed going forwards.

Reply:
removed after offline discussion